internal/apigw/apigw_openapi.go

```
@@ -474,6 +480,93 @@ func getTerraformEntityOperationExtension(operation *apigw_v1.Operation) *yaml.N
 	return extensionNode
 }
 
+func getTerraformEntityOperationPagination(operation *apigw_v1.Operation) *yaml.Node {
+	if operation.Pagination == nil {
+		return nil // Skip if no pagination is defined
+	}
+
+	paginationNode := &yaml.Node{
+		Kind:    yaml.MappingNode,
+		Content: []*yaml.Node{},
+	}
+
+	// type
+	paginationType := ""
+	switch operation.Pagination.Type {
+	case apigw_v1.Pagination_TERRAFORM_ENTITY_PAGINATION_TYPE_UNSPECIFIED:
```

is it better to throw an error? Looks like pagination doesn't work at all if type is `unspecified`.